compiler/src/iree/compiler/Codegen/Common/TileDispatchUsingForall.cpp

```diff
@@ -53,9 +53,17 @@ struct TilingInfo {
 static FailureOr<TilingInfo>
 getTiledAndDistributionInfo(RewriterBase &rewriter,
                             ArrayRef<Operation *> computeOps) {
+  // TODO(Max191): We choose the root for tiling as the first op that has a
+  // lowering config with workgroup tile sizes. It is probably cleaner to
+  // reuse some logic for finding the root operation during lowering strategy
+  // selection, but root op selection logic would need to either be unified or
+  // passed as an option to this pass.
```

Comment:
IMO this TODO comment is not needed. The way that the tiling root is found here makes the most sense to me, and strategy selection just has to make sure it only gives workgroup tile sizes to the root op.

Reply:
To me the main reason to do this would be to make the connection to root selection more clear, but thinking about it some more, I think we actually don't want this to be coupled with root op selection. The main benefit I thought that would achieve is to make the strategy selection more foolproof, but I think it is better to do that with some verification step in `TileDispatchUsingForall`.